app/views/userProfile.scala.html

```diff
@@ -163,6 +163,7 @@ <h3>@Messages("validations")</h3>
         <div id="mistakes-row" class="row">
             <h2>@Messages("dashboard.mistakes.header")</h2>
             <span id="mistakes-subheader"></span>
+            <span id ="no-mistakes-subheader"></span>
```

Comment:
I don't see why we have two subheaders now. If I'm reading your code correctly, we either fill one with text or the other one with text. So I don't see a reason to have an extra HTML element lying around! It seems like we should just set the text for a single subheader.